### PR TITLE
media-gfx/librecad: version bump: 2.0.8

### PR DESCRIPTION
**Pull request by dxli**
in atomic commits this time.

### CODE REVIEW

File: media-gfx/librecad/librecad-2.0.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+S="${WORKDIR}/LibreCAD-${PV}"
+
+src_prepare() {
+	# currently RS_VECTOR3D causes an internal compiler error on GCC-4.8
+	use 3d || sed -i -e '/RS_VECTOR2D/ s/^#//' librecad/src/src.pro || die
+}
+
+src_configure() {
+	    if use qt4; then
+	            eqmake4 -r
```
Comment (hasufell) [inline media-gfx/librecad/librecad-2.0.8.ebuild]: indenting looks broken here

File: media-gfx/librecad/librecad-2.0.8.ebuild [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+			else
+	            eqmake5 -r
+	    fi
+}
+
+src_install() {
+	dobin unix/librecad
```
Comment (hasufell) [inline media-gfx/librecad/librecad-2.0.8.ebuild]: please make a few newlines for readability, it's hard to follow such a block

**Comment by dxli** [comment]
cleanup of indentation

**Comment by dxli** [comment]
updated to allow auto merging

**Comment by dxli** [comment]
thanks!
